Scripts/update-sprite.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
We already have powershell and python scripts, I would like to avoid adding another language to the scripts if possible...